python/tests/mlir/ast_elif.py

```diff
@@ -28,7 +28,9 @@ def cost(thetas: np.ndarray):  # can pass 1D ndarray or list
                 rx(theta, q[i % 4])
 
     print(cost)
-    cost(np.asarray([1., 2., 3., 4., 5., 6.]))
+    ## [SKIP_TEST] : Following gives as error on 'python/cudaq/kernel/utils.py:176: ValueError'
+    ## ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
+    # cost(np.asarray([1., 2., 3., 4., 5., 6.]))
```

Comment:
@amccaskey : Is this a regression?

Reply:
@bmhowe23  is this the same as Marwa's recent error?

Reply:
Fix at #1406

Reply:
Yes, I think #1370 introduced this issue.

Reply:
These tests were not running in CI, so the regression escaped, I've enabled them in this PR...

Reply:
I will update this if PR https://github.com/NVIDIA/cuda-quantum/pull/1406 is merged before this, or I can wait...